ansible-ipi-install/roles/node-prep/templates/install-config.j2

```diff
@@ -20,6 +20,9 @@ platform:
     dnsVIP: {{ dnsvip }}
 {% if (release_version[0]|int > 4) or ((release_version[0]|int == 4) and (release_version[2]|int > 3)) %}
     provisioningNetworkInterface: {{ prov_nic }}
+{% endif %}
+{% if bootstraposimage is defined and bootstraposimage|length %}
+    bootstrapOSImage: {{ bootstraposimage }}
```

Comment:
To be consistent with the variable used in the documentation, can we change it from 'bootstraposimage' to 'bootstrapOSImage' ?

Reply:
only if you change
```
baseDomain: {{ domain }}
name: {{ cluster }}
machineCIDR: {{ extcidrnet }}
networkType: {{ network_type }}
apiVIP: {{ apivip }}
ingressVIP: {{ ingressvip }}
dnsVIP: {{ dnsvip }}
...
```

Reply:
haha fine you win